.github/workflows/build-publish.yml

```diff
@@ -18,9 +18,17 @@ jobs:
       - uses: actions/setup-node@v1
         with:
           node-version: 13
-      - run: yarn install --frozen-lockfile
-      - run: yarn lint
-      - run: yarn boot
+      - uses: actions/cache@v2
```

Comment:
Nå har vi basically denne koden fire steder: 
```
    - uses: actions/cache@v2
      with:
        path: '**/node_modules'
        key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
    - name: Install deps
      if: steps.yarn-cache.outputs.cache-hit != 'true'
      run: yarn install --frozen-lockfile
    - name: Gulp-build
      run: yarn boot
```
Det er noen små forskjeller som at linting skjer mellom install og build i denne workflowen, og at en annen ikke kjører `yarn boot`, men heller `yarn gulp:build` direkte. Jeg vet ikke om de forskjellene er bevisste eller ikke. Om de ikke er det så er nok ikke det så heldig. Kan det gi mening å gjenbruke denne koden, og er det en enkel måte å oppnå det?

Reply:
Er ingen enkel måte å gjenbruke koden med et call, men kan se på løsninger som bruker matrix strategy: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategy